projects/packages/my-jetpack/_inc/components/products-table-view/style.scss

```diff
@@ -0,0 +1,91 @@
+@import '@wordpress/dataviews/build-style/style.css';
+@import '@wordpress/components/build-style/style.css';
+
+:root {
+    --font-size: 13px;
```

Comment:
We usually use 16 as the base font-size and, comparing the mockups with the actual implementation, it looks smaller than in the mockups. Checking the values there, it also seems different. Did that change somewhere?

Reply:
This was actually only updating the font size of the CTAs, the font sizes of the table itself come from the dataviews component.